articles/flutter-vs-reactnative/index.md

```diff
@@ -0,0 +1,98 @@
+
+##### Software architecture and state management
+React native has a mature ecosystem around this. The developers have the freedom to choose which navigation package and global state management package they want without them worrying about how widely adopted it is.
+
+Flutter has only a handful of such packages, with each package not having widespread knowledge. For example, I use a package called stacked for state management, but it was hardly known in the community until now. This problem will surely go away in the future, with the rising adoption of Flutter as a framework.
+
+##### App sizes
+One of the bigger tradeoffs with the cross-platform is the app size, it is bound to be larger than native apps. But in this respect, it is also usually seen that React apps are relatively smaller than Flutter apps.
+
+##### Documentation
+React has decent documentation. The only problem is that it is more focused towards the web developers and not app developers. Flutter’s 280-page documentation, is one of the best documentation you will ever find. Flutter also has a very active YouTube Channel, showcasing tips and tricks.
+
+##### Code reusability
+You cannot run React Native code directly on browsers or desktop. What you can do though is reuse parts of the code between let’s say React Native and React web. Flutter, on the other hand, allows you to use the same code and compile builds for whatever platforms it supports. This, of course, is dependent if all 3rd party dependencies are platform-agnostic or not. You just need to adjust the UI for various screen orientations!!
+
+##### CI/CD support
+React Native documentation does not have anything regarding CI/CD. You will have to use a third-party service like Fastlane or Codemagic to automate the deliveries. Flutter has a well-covered documentation and a rich CLI tool as well.
+
+##### The Future?
+Just a week ago Flutter announced an alpha release of Flutter for Windows. This means you can now make UWP apps with the Flutter framework. So the same app that runs on your iPad can also run on an Xbox or even a Hololens for that matter, so cool!
+
+The list of possibilities just keeps on increasing. Still, it is quite clear that Flutter is playing catch up React, but it’s getting there fast.
+React has maturity as an ecosystem. It also has the bases covered in terms of platforms supported.
+
+What they are doing is working on an ambitious project called Fabric. It is rewiring the JS framework from the ground up, aiming to make native API calls easier, support async rendering, and new threading models.
```

Comment:
It may be helpful to include a link regarding fabric

Reply:
Added a blogpost related to fabric and why it is exciting.